evaluate.py

```diff
@@ -171,21 +170,39 @@ def run_evaluation_episode(trace_name, model, world, camera, car, video_dir, sav
 
     return run_metrics
 
+
+def create_steering_model(args):
+    if args.model_type == "steering":
+        model = SteeringModel(args.model)
+    elif args.model_type == "conditional-steering":
+        model = ConditionalSteeringModel(args.model, args.speed_model)
+    elif args.model_type == "conditional-waypoints":
+        model = ConditionalWaypointsModel(args.model, args.speed_model)
```

Comment:
Currently there is no way to override the default arguments for these models. Maybe that's OK now, but in the long run, it would be helpful to have a scalable way to do it without adding dozens of arguments into the evaluation script. A yaml model config file? 🤔

Reply:
I think this can be left as a future work.

Reply:
In the long run yaml file would indeed be better than a long list of bash parameters.